scripts/run_integration_tests_aarch64.sh

```diff
@@ -311,29 +309,29 @@ echo 6144 | sudo tee /proc/sys/vm/nr_hugepages
 sudo chmod a+rwX /dev/hugepages
 
 # Run all direct kernel boot (Device Tree) test cases in mod `parallel`
-time cargo test $features "parallel::$test_filter" -- ${test_binary_args[*]}
+time cargo test $features "parallel::$test_filter" --target $BUILD_TARGET -- ${test_binary_args[*]}
```

Comment:
I think there is no need to build the cargo test binaries with `musl`, as long as the `cloud-hypervisor` binary is built to the correct target (which is already done in this script).

Reply:
I thought like that, too. But an error was seen in linking the test binary, if the same `target` is missing:

`  = note: /usr/bin/ld: cannot find libgcc_s.so.1
          collect2: error: ld returned 1 exit status
`